Source/HelixToolkit.Wpf.SharpDX/Model/Elements3D/PointGeometryModel3D.cs

```diff
@@ -190,19 +190,20 @@ private void OnColorChanged()
         /// <param name="host"></param>
         public override void Attach(IRenderHost host)
         {
-            renderTechnique = Techniques.RenderPoints;
+            renderTechnique = host.RenderTechniquesManager.RenderTechniques[DefaultRenderTechniqueNames.Points];

```

Comment:
All `Model3D` objects now get access to the dictionary of `RenderTechniques` from their `IRenderHost`.